db/sqitch.plan

```diff
@@ -73,8 +73,8 @@ tables/ccbc_user_001_no_unique_email 2022-10-26T19:00:26Z Matthieu Foucault <mat
 tables/application_001_add_analyst_permission 2022-10-24T14:56:56Z Anthony Bushara <anthony@button.is> # Adds the analyst permissions and RLS for the application table
 tables/application_status_001_give_read_and_insert_to_ccbc_analyst 2022-10-24T17:29:23Z Anthony Bushara <anthony@button.is> # Gives the select and insert permission to ccbc_analyst
 computed_columns/application_project_name [computed_columns/application_project_name@1.9.0] 2022-10-24T19:27:49Z Anthony Bushara <anthony@button.is> # Add ccbc_analyst to application_project_name computed column
-computed_columns/application_status [computed_columns/application_status@1.9.0] 2022-10-24T19:30:15Z Anthony Bushara <anthony@button.is> # Add ccbc_analyst to application_status computed column
```

Comment:
Why was the sqitch plan changed here and a rework done? This change is already reworked in this release, it doesn't need to be reworked again

Reply:
I pointed out that it was failing the `check_immutable_sqitch_files` and assumed it needed a sqitch add or rework.

Reply:
Looks like this change was added in #815, which was merged after this PR was initially created.

Reply:
So we have a proper conflict here, where two PRs made similar changes: reworking `computed_columns/application_status`. This PR can now modify `computed_columns/application_status.sql` directly without altering the sqitch plan, or creating the `@1.11.0.sql` files

Reply:
hmm.. #815 added `computed_columns/application_status [computed_columns/application_status@1.9.0]` so it's ok for current PR to add `computed_columns/application_status [computed_columns/application_status@1.11.0]` files. Otherwise we'll get  check_immutable_sqitch_files  error

Reply:
`computed_columns/application_status` isn't immutable right now since it's been reworked in #815 and the reworked change is unreleased. You shouldn't get the  `check_immutable_sqitch_files` error.